CODEOWNERS

```
@@ -0,0 +1,2 @@
+# release team
+pyproject.toml  @sksat
```

例によって私も書いておかないと困ることありそう
```suggestion
pyproject.toml  @sksat @KOBA789 
```